DEVELOPERS.md

```diff
@@ -8,12 +8,28 @@ Follow [the NCTR's Python Style Guide](https://github.com/NationalCentreTruthRec
 
 ## Linting + Formatting
 
-Before beginning development, please configure git to use the git hooks (`.git/hooks`) provided in the repository. This will ensure that the linting configuration is updated before each commit.
+Before beginning development, please configure git to use the git hooks (`.git/hooks`) provided in the repository. This will ensure that the linting configuration for Python is updated before each commit.
 
 ```bash
 git config core.hooksPath .githooks
 ```
 
+Additionally, install the [ESLint VSCode extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and add this configuration to your [local VSCode settings.json](https://code.visualstudio.com/docs/getstarted/settings#_settings-json-file) for JavaScript linting:
+
+```json
+{
+    "[javascript]": {
+        "editor.formatOnSave": false,
+        "editor.codeActionsOnSave": {
+            "source.fixAll.eslint": "explicit"
+        },
+        "editor.rulers": [
+            99
+        ]
+    }
+}
+```
+
```

Comment:
Can you add a note to run `npm install --include=dev` here?

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/287/commits/184b98bcafdc2623cced22cdb22c701837979ce9